pkg/devfile/parser/data/versions.go

```diff
@@ -15,6 +15,7 @@ type supportedApiVersion string
 const (
 	APIVersion200 supportedApiVersion = "2.0.0"
 	APIVersion210 supportedApiVersion = "2.1.0"
+	V2APIVersion  supportedApiVersion = "v1alpha2"
```

Comment:
It is because kubernetes CRD does not have schema version, it only contains APIVersion, which is in the format of `group/apiversion`. 
for example: APIVersion: "workspace.devfile.io/v1alpha2" uses api version v1alpha2, and match to v2 schema

should use the latest v2 schema version if the apiversion is `v1alpha2`. as `v1alpha2` won't be bumped up unless breaking changes have been introduced

Reply:
updated